src/main/networking/vtunnel.ts

```diff
@@ -31,24 +43,70 @@ export class vtunnel {
     },
   });
 
-  async generateVtunnelConfig() {
+  // eslint-disable-next-line no-useless-constructor
+  private constructor() {}
```

Comment:
There's no need for a private constructor — nobody else can even get at the class to call `new` on it.
(Well, they _could_ by using `Object.getPrototypeOf(getVtunnelInstance())`, but in that case TypeScript `private` markings wouldn't help.)